packages/ui/src/utils/numberHelpers.ts

```diff
@@ -134,3 +135,35 @@ export function formatDecimalValues(value?: string, decimals?: number): string {
   const decimalVal = decimals === undefined ? 4 : decimals;
   return value ? parseFloat(value).toFixed(decimalVal) : "0";
 }
+
+const BILLION = "1000000000";
+const MILLION = "1000000";
+const THOUSAND = "1000";
+
+/**
+ * @param value A numeric string to format.
+ * @returns The formatted numeric string and the suffix (K, M, B) to make large numbers more readable.
+ */
+export const formatAbbreviatedNumber = ({
```

Comment:
There is an existing `formatNumberAsAbbr`. Can we consolidate/clean-up these functions?